aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -395,6 +396,15 @@ static void clamp_min_kernel(TensorIterator& iter, Scalar min_scalar) {
   });
 }
 
+static void kaiser_window_kernel(TensorIterator& iter, int64_t window_length, double beta){
+  AT_DISPATCH_FLOATING_TYPES_AND(kBFloat16, iter.dtype(), "kaiser_window_cpu", [&](){
+    auto alpha = (window_length - 1) / 2.0;
```

Comment:
This is a little misleading since it will always be double. That means the computation will always occur in double, too. Is that what's intended?

Reply:
No, I forgot to cast, but I'll be fixing this.